### ISSUE TITLE
Text component sometimes cause a malfunction

### ISSUE THREAD

**Issue by paiext** [pureqml/qmlcore#173]
The following codes cause the app to become unresponsive:
If you change myText.text to "Foo," the app will run normally.

```
Item {  
  Rectangle {
    width: 295;
    height: 452;    
    color: 'black';
    
    Rectangle {
      width: parent.width; 
      height: 100;
      
      Item {
        width: 30;
        height: 100;
        Text {          
          anchors.horizontalCenter: parent.horizontalCenter;
          anchors.verticalCenter: parent.verticalCenter;
          font.pixelSize: 12;
          font.family: "Tahoma";          
          text: 'Hue';
        }
      }    
    } 
  }  
}
```

**Comment by whoozle** [comment]
removed offending ancient hack in c413e7eeaf2b394114fe9be51c8497a39216bc4d

This has to me reevaluated and re-fixed properly

**Comment by whoozle** [comment]
not so fast :-1: it's broken the same way.

**Comment by whoozle** [comment]
fixed in e96e23b38f510725be900fb4c2935661f440526c
